docs/cep/stream-workers-sdk-example.md

```diff
@@ -308,164 +313,176 @@ print("Activate", client.activate_stream_app('Sample-Cargo-App', True))
-    /**
-    Testing the Stream Application:
-        1. Open Stream SampleCargoAppDestStream in Console. The output can be monitored here.
-        2. Upload following data into SampleCargoAppInputTable C8DB Collection
-            {"weight": 1}
-            {"weight": 2}
-            {"weight": 3}
-            {"weight": 4}
-            {"weight": 5}
-        3. Following messages would be shown on the SampleCargoAppDestStream Stream Console
-            [2021-08-27T14:12:15.795Z] {"weight":1}
-            [2021-08-27T14:12:15.799Z] {"weight":2}
-            [2021-08-27T14:12:15.805Z] {"weight":3}
-            [2021-08-27T14:12:15.809Z] {"weight":4}
-            [2021-08-27T14:12:15.814Z] {"weight":5}
-    */
-    -- Create Table SampleCargoAppInputTable to process events.
-	CREATE SOURCE SampleCargoAppInputTable WITH (type = 'database', collection = "SampleCargoAppInputTable", collection.type="doc", replication.type="global", map.type='json') (weight int);
+@App:qlVersion("2")
+@App:description('Basic stream worker to demonstrate reading data from input stream and store it in a collection. The stream and collections are automatically created if they do not already exist.')
+/**
+    Test the stream worker:
+    1. Open Stream SampleCargoAppDestStream in Console. The output can be monitored here.
+    2. Upload following data into SampleCargoAppInputTable collection:
+        {"weight": 1}
+        {"weight": 2}
+        {"weight": 3}
+        {"weight": 4}
+        {"weight": 5}
+    3. Following messages are shown on the `SampleCargoAppDestStream` Stream Console:
+        [2021-08-27T14:12:15.795Z] {"weight":1}
+        [2021-08-27T14:12:15.799Z] {"weight":2}
+        [2021-08-27T14:12:15.805Z] {"weight":3}
+        [2021-08-27T14:12:15.809Z] {"weight":4}
+        [2021-08-27T14:12:15.814Z] {"weight":5}
+    4. Following messages are stored into SampleCargoAppDestTable
+        {"weight":1}
+        {"weight":2}
+        {"weight":3}
+        {"weight":4}
+        {"weight":5}
+*/
 
-    -- Create Stream SampleCargoAppDestStream
-	CREATE SINK SampleCargoAppDestStream WITH (type = 'stream', stream = "SampleCargoAppDestStream", replication.type="local") (weight int);
+-- Defines Table SampleCargoAppInputTable
+CREATE SOURCE SampleCargoAppInputTable WITH (type = 'database', collection = "SampleCargoAppInputTable", collection.type="doc", replication.type="global", map.type='json') (weight int);
 
-    -- Data Processing
-    @info(name='Query')
-    INSERT INTO SampleCargoAppDestStream
-    SELECT weight
-    FROM SampleCargoAppInputTable;
-    """
-# Validate a stream application
+-- Define Stream SampleCargoAppDestStream
+CREATE SINK SampleCargoAppDestStream WITH (type = 'stream', stream = "SampleCargoAppDestStream", replication.type="local") (weight int);
+
+-- Defining a Destination table to dump the data from the stream
+CREATE STORE SampleCargoAppDestTable WITH (type = 'database', stream = "SampleCargoAppDestTable") (weight int);
```

Comment:
Let's make this definition more simple:
```
CREATE TABLE SampleCargoAppDestTable (weight int);
````

Reply:
@StoyanTc can you check my comment on https://macrometa.atlassian.net/browse/CHG-605. This is relevant to this example and all examples where we are not specifying `replication.type` in `stream workers` Since CREATE STORE, CREATE TABLE without specifying the replication type will a `local `collection and we have removed support for local collections from play and paas and might remove support other federation as well..

Reply:
@StoyanTc, would you mind either adding the change as a suggestion or putting it in as a commit? I'm not super confident in my command of stream workers or JavaScript.

Btw, if you want to change this example, it's used several times in the docs and I think it is also in the sample apps in the product.

Reply:
@StoyanTc, you say you tested the full demo, but did you also test the per-step method I explained in the comment?

Reply:
@diana-macrometa - I don't have rights to make changes here. 
There are two reasons for the issue you have with the ad-hock query in step 5:
1. The original sample `Sample-Cargo-App` does not have store/table `SampleCargoAppDestTable`.
2. The created stream worker is not activated/published.